Resfeber/View Controllers/TripDetailViewController.swift

```diff
@@ -302,7 +312,52 @@ extension TripDetailViewController: UITableViewDelegate {
         let name = placemark.name
         let latitude = placemark.location?.coordinate.latitude
         let longitude = placemark.location?.coordinate.longitude
-        tripsController.addEvent(name: name, latitude: latitude, longitude: longitude, trip: trip)
+        let address = placemark.address
+        tripsController.addEvent(name: name, latitude: latitude, longitude: longitude, address: address, trip: trip)
         reloadTrip()
     }
 }
+
+extension TripDetailViewController: MKMapViewDelegate {
+    func mapView(_ mapView: MKMapView, viewFor annotation: MKAnnotation) -> MKAnnotationView? {
+        guard let event = annotation as? Event else { return nil }
+        
+        guard let annotationView = mapView.dequeueReusableAnnotationView(withIdentifier: Event.annotationReuseIdentifier, for: event) as? MKMarkerAnnotationView else {
+            NSLog("Missing the registered map annotation view")
+            return nil
+        }
```

Comment:
```
        guard let event = annotation as? Event,
         let annotationView = mapView.dequeueReusableAnnotationView(withIdentifier: Event.annotationReuseIdentifier, for: event) as? MKMarkerAnnotationView else {
            NSLog("Missing the registered map annotation view")
            return nil
        }
```

Reply:
Implemented the above suggestion in a followup commit